tests/src/core/docsUrl.js

```diff
@@ -1,13 +1,14 @@
 import { expect } from 'chai'
 
+import pkg from '../../../package'
```

Comment:
also here :-)

Reply:
Yea.. realised it after the first push 😅. Done.